UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;
+  if (value == -1)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_exchange_rate_loading"
+        small
+        withOverlay={false}
+      />
+    );
+  return (
+    <div>
+      {loadingIcon}
+      <span className="AppHeader_exchange_rate">
+        {value == -1 ? `` : `$${value}`} <span className="bold">USD/MYB</span>

```

Comment:
Do we want to show an empty value? Probably we could do `{ value && '$${value}'}` (using single quote comma to format comment in Github, use inverse tilde instead). So what this does is checks if `value` is set, and then if it does, displays it as we needed formatted.

Reply:
fixed